docs/user_stories/loadflow_tutorial.md

```diff
@@ -0,0 +1,65 @@
+---
+title: User story number 1
+layout: default
+---
+
+# Easy network to start with PowSyBl
+
+In this user story, the network is quite simple and made of two lines in parallel, with on left side a generator and on right side a load. The load consumes 600 MW and the generator produces 606.5 MW. A load flow computes the flows on the two lines. The flow on the upper line is of 302.4 MW at its entrance and of 300.4 MW at its exit. The flow in the lower line is the same. The power losses are of 2 MW on each line.   
```

Comment:
You both use "the flow **in** the line" and "the flow **on** the line". I think the correct locution is "the flow **through** the line"